app/models/users/student.rb

```diff
@@ -14,4 +18,17 @@ def self.is_student?(data)
     Student.where(email: data["email"]).first
   end
 
+  def self.lost
+    result = []
+    result << joins(events: [:scans] ).where(at_school: true)
+                                      .where("start_time <= ? AND end_time >= ?", Time.now, Time.now )
+                                      .where.not(scans:{correct: true})
+    joins(:events).where(at_school: true)
+                  .where("start_time <= ? AND end_time >= ?", Time.now, Time.now).each do |student|
+                    result << student if student.events.last.scans.count == 0
+                  end
+
+    result.flatten
```

Comment:
@jedschneider Currently I'm testing it as an integration test in `admin/grove_monitor_spec` when you click on lost or absent, we send a filter parameter which brings us back to the `absent` scope and `lost` method within the student model. You're right that they're could be more unit tests in the `student_spec` I'm gonna be working on this today.